packages/extension-browser/src/devtools/views/controls/settings.css

```diff
@@ -0,0 +1,3 @@
+.root {
+    margin-top: 1.5rem;
```

Comment:
Could we achieve this with a grid and gap in the parent component (or at least with a passed through class name)? I don't like having child-components control layout outside of their contents.